Gajeongtongsin/Gajeongtongsin/Screens/Parent/Sending/SentMessageListViewController.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  SentMessageListViewController.swift
+//  Gajeongtongsin
+//
+//  Created by Youngwoong Choi on 2022/07/22.
+//
+
+import UIKit
+
+class SentMessageListViewController: BaseViewController {
+    //MARK: - Properties
+    let messageList: [Message] = mainTeacher.parentUserIds.flatMap({$0.sendingMessages})
+//        .filter({$0.type != .emergency})
+    
+    //화면에 뿌려줄 메시지 리스트를 곧바로 'messageList#'으로 지정하지 않고, 부모 유저(여기선 parent1)에 속한 것으로 불러옴
+    let currentParent = parent1
```

Comment:
```suggestion
  var currentParent : ParentUser {
        return mainTeacher.parentUserIds[0]
    } 
```

computed 속성으로 작성을 하면 테이블 리프레쉬하는 문제에 도움이 되지 않을까요?😁

Reply:
DB 에서 티처 하나만 정하면 줄줄이 따라 불러오기로 해놓고..
결국 MVP에서는 티처가 한 명이니까 부모 중에서만 고른다고 생각한 것 같습니다 ㅎㅎ

알려주신대로 수정했습니다.